hamocc/mo_parambgc_ini.F90

```diff
@@ -0,0 +1,1241 @@
+    !     Radiation and attenuation parameters
+    !********************************************************************
+    ! parameters for sw-radiation attenuation
+    ! Analog to Moore et al., Deep-Sea Research II 49 (2002), 403-462
+    ! 1 kmolP = (122*12/60)*10^6 mg[Chlorophyl] 
+    ctochl  = 60.        ! C to Chlorophyl ratio
+    atten_w = 0.04       ! yellow substances attenuation in 1/m
+    atten_c = 0.03*rcar*(12./ctochl)*1.e6  ! phytoplankton attenuation in 1/m 
+    atten_uv= 0.33       ! 
+    atten_f = 0.4        ! fraction of sw-radiation directly absorbed in surface layer 
+                         ! (only if FB_BGC_OCE) [feedback bgc-ocean]
+
+    !********************************************************************
+    !     Dust deposition and iron solubility parameters
+    !********************************************************************
+    !ik weight percent iron in dust deposition times Fe solubility
+    ! the latter three values come from Johnson et al., 1997
+    fetune     = 0.6                  ! factor introduced to tune deposition/solubility
+    perc_diron = fetune * 0.035 * 0.01 / 55.85
+    fesoly     = 0.5*1.e-9            ! max. diss. iron concentration in deep water 
+    relaxfe    = 0.05/365.            ! 1/d complexation rate to relax iron concentration to fesoly
+
+    !********************************************************************
+    !     Sinking parameters
+    !********************************************************************
+    wpoc   =  5.       !m/d   Sinking speed of detritus iris : 5.
+    wcal   = 30.       !m/d   Sinking speed of CaCO3 shell material
+    wopal  = 30.       !m/d   Sinking speed of opal iris : 60
+#if defined(WLIN) && ! defined(AGG)
+    wmin   =  1.       !m/d   minimum sinking speed
+    wmax   = 60.       !m/d   maximum sinking speed
+    wlin   = 60./2400. !m/d/m constant describing incr. with depth, r/a=1.0
+#endif
+#ifndef AGG
+    dustd1   = 0.0001 !cm = 1 um, boundary between clay and silt
+    dustd2   = dustd1*dustd1
+    dustsink = (9.81 * 86400. / 18.                  &  ! g * sec per day / 18.
+     &         * (claydens - 1025.) / 1.567 * 1000.    &  !excess density / dyn. visc.
+     &         * dustd2 * 1.e-4)
+    wdust = dustsink
+#endif    
+  end subroutine
+
+  !---------------------------------------------------------------------------------------------------------------------------------
+  subroutine read_bgcnamelist()
+    !
+    ! Read the bgcparams namelist for parameter tuning.
+    ! Note that afterward, i) rates need to be adjusted for timestep
+    ! and some depending parameters need re-calculation
+    !
+
+    integer  :: iounit
+
+    namelist /bgcparams/ bkphy,dyphy,bluefix,bkzoo,grazra,spemor,gammap,gammaz,ecan,zinges,epsher,bkopal,rcalc,ropal,              &
+                       & remido,drempoc,dremopal,dremn2o,dremsul,fetune,relaxfe,wpoc,wcal,wopal
```

Comment:
`wpoc` would only be used directly if the old HAMOCC sinking (constant wpoc) is used. Shouldn't we rather include wmin/wmax/wlin of the WLIN scheme as tuning parameters? (I would even be tempted to remove the constant sinking  completely from HAMOCC, since this produces crap results)

Reply:
I am totally happy to extend the parameter list of the namelist. So far, I saw the new namelist as extendable and only for relevant parameters.

Reply:
Done. I left the constant sinking, though - can be discussed afterward in an extra PR.